datadog_checks_base/datadog_checks/base/checks/win/wmi/sampler.py

```diff
@@ -374,7 +374,7 @@ def build_where_clause(fltr):
             while f:
                 prop, value = f.popitem()
 
```

Comment:
can we log an error message if the filter is a tuple.

Reply:
Resolving after our discussion today